zephyr/Kconfig

```diff
@@ -33,4 +33,7 @@ config BOOT_SIGNATURE_KEY_FILE
 	  with the public key information will be written in a format expected by
 	  MCUboot.
 
+config FLASH_BLOCK_ALIGN
+	def_int $(dt_int_val,DT_FLASH_WRITE_BLOCK_SIZE)
```

Comment:
Could we give it a name that is as similar as possible to the source? Then it would be easier for users to understand that this is the same thing.

e.g.

CONFIG_DT_FLASH_WRITE_BLOCK_SIZE

Reply:
Done